### PR TITLE
 fix hints migration excessive spaces

### PR DESCRIPTION
**Pull request by dandimeo**
excessive spaces in hints content will markdownify the entire content as codeblock.
Removed all extra spaces from the migrated hints.
Regenerated content.

### CODE REVIEW

**Review by Simran-B — COMMENTED** [review]
Does this preserve content like fenced codeblocks with indentation inside of hint boxes, like this one?
https://github.com/arangodb/docs/pull/1256/files

**Comment by dandimeo** [comment]
@Simran-B 
Yes it preserves the indentation in codeblocks, this is taken from the latest migration, the content you see will actually change when the DOC-349 related pull request will modify the hints migration, too. (In the image there are still the old tabs shortcodes etc.)
I would like to close and merge this pull request and shift the validation of hints in the DOC-349 pull request.

File: migration-tools/utils.py [hunk 1]
```diff
@@ -32,19 +32,25 @@ def migrate_hints(paragraph):
...
 
     return paragraph
 
 def migrate_capture_alternative(paragraph):
     captureRE = re.findall(r"(?<={% capture alternative %})(.*?)(?= {% endcapture %})", paragraph, re.MULTILINE | re.DOTALL)
     for capture in captureRE:
-        info = f"{{{{% hints/info %}}}}\n{capture}\n{{{{% /hints/info %}}}}"
+        info = f'\n\
+{{{{% hints/info %}}}}\n\
+  {capture}\n\
```
Comment (Simran-B) [inline migration-tools/utils.py:52]: What are the two leading spaces for?
```suggestion
{capture}\n\
```

**Review by Simran-B — CHANGES_REQUESTED** [review]
There appear to be two blank lines above the starting `{{% hints... %}}`, but it should only be one.

File: migration-tools/utils.py (hunk 1 above)
Comment (dandimeo) [reply migration-tools/utils.py]: Changed in [2d9c94a](https://github.com/arangodb/docs-hugo/pull/5/commits/2d9c94ab3f696535e35b9664b2de75dc76b56954)

**Review by Simran-B — APPROVED** [review]

**Pull request merged**